src/css_parser.rs

```diff
@@ -1563,6 +1611,13 @@ pub enum CssDirectionParseError<'a> {
     CornerError(CssDirectionCornerParseError<'a>),
 }
 
+impl_display!{CssDirectionParseError<'a>, {
+    Error(e) => e,
+    InvalidArguments(val) => format!("Invalid arguments: \"{}\"", val),
+    ParseFloat(e) => format!("Invalid value: {}", e),
```

Comment:
Use `\"{}\"` instead of `{}` when displaying values for consistency.

Reply:
There is no value displayed.

Implementation for `ParseFloat::Display`: https://doc.rust-lang.org/beta/src/core/num/dec2flt/mod.rs.html#175

Reply:
Okay, then leave it as it is. Is weird though, I thought the `ParseFloat` also carried the CSS string.